src/main/java/it/eng/idsa/multipart/domain/MultipartMessage.java

```diff
@@ -16,126 +14,89 @@
 /**
  * Type of the MultipartMessage.
  */
-public class MultipartMessage {
-	
-	private Map<String, String> httpHeaders = new HashMap<>();
-	private Map<String, String> headerHeader = new HashMap<>();
-	private Message headerContent = null;
-	private Map<String, String> payloadHeader = new HashMap<>();
-	private String payloadContent = null;
-	private Map<String, String> signatureHeader= new HashMap<>();
-	private String signatureContent = null;
-	private String token = null;
-	
-	public MultipartMessage() {
-		super();
-	}
+public class MultipartMessage implements AutoCloseable{
 	
+	private static final Cleaner cleaner = Cleaner.create();
```

Comment:
Not sure if formatting is not rendered in browser correct or it needs to be applied.

Reply:
In my IDE it was ok, but I also see it in the browser more indent. I will try to fix it somehow